influxql/iterator.gen.go

```diff
@@ -767,7 +767,7 @@ func (itr *floatInterruptIterator) Next() (*FloatPoint, error) {
 	if itr.count&0xFF == 0xFF {
 		select {
 		case <-itr.closing:
-			return nil, nil
+			return nil, itr.Close()
```

Comment:
This is the code path where a query is interrupted either via `kill query` or a hitting a limit.  The cause of #7811 is that `itr.Close()` is not called in this case.  I can change it to call `itr.Close()` and return nil though.